hera_qm/utils.py

```diff
@@ -326,6 +356,8 @@ def get_metrics_ArgumentParser(method_name):
         ap.add_argument("--run_if_first", default=None, type=str, help='only run \
                         day_threshold_run if the first item in the sorted data_files \
                         list matches run_if_first (default None means always run)')
+        ap.add_argument("--skip_making_flagged_abs_calfits", default=False, action="store_true",
+                        help='If True, flag the abscal files.')
```

Comment:
This help string has things backwards, I think

Reply:
Thanks for catching this!